scio-test/src/test/scala/com/spotify/scio/testing/JobTestTest.scala

```diff
@@ -296,6 +310,26 @@ class JobTestTest extends PipelineSpec {
     an [AssertionError] should be thrownBy { testDistCacheJob("a1", "a2", "b1", "b2", "c3", "d4") }
   }
 
+  def testCustomIOJob(xs: Int*): Unit = {
+    JobTest[CustomIOJob.type]
+      .args("--input=in.txt", "--output=out.txt")
+      .input(CustomIO("TextIn"), Seq(1, 2, 3))
```

Comment:
I know. It's mostly for aesthetic reason since, i.e. avoid multi-line function argument.